src/main/java/org/toradocu/translator/preprocess/ImplicitParamSubjectPatterns.java

```diff
@@ -21,45 +21,56 @@
   public String run(BlockTag tag, DocumentedExecutable excMember) {
     String comment = tag.getComment().getText();
     String parameterName = ((ParamTag) tag).getParameter().getName();
-    String[] patterns = {
-      "must be",
+    String[] positivePatterns = {"must be", "will be", "Will be", "Must be"};
+
+    String[] negativePatterns = {
```

Comment:
Patterns starting with capital letters are different from the ones starting with lowercase letters. Is there a particular reason for that?

Reply:
No reason apparently, we never noticed before - fixed